### PR TITLE
Run more sizes. Make output more compact

### PR DESCRIPTION
**Pull request by kronbichler**
This also prints the number of equivalent iterations of the s-step method, i.e., the number of matrix-vector products.

### CODE REVIEW

File: common_code/benchmark.h [hunk 1]
```diff
@@ -243,16 +258,17 @@ run_templated(const unsigned int s, const bool short_output, const MPI_Comm &com
...
+                << " " << std::setw(10) << tria.n_global_active_cells()                      //
+                << " " << std::setw(11) << dof_handler.n_dofs()                              //
+                << " " << std::setw(9) << solver_time / n_iterations                         //
+                << " " << std::setw(10) << dof_handler.n_dofs() / solver_time * n_iterations //
+                << " " << std::setw(4) << n_iterations                                       //
+                << " " << std::setw(9) << matvec_time;                                       //
 
       for (const auto &t : profile)
-        std::cout << " | " << std::setw(11) << t / n_iterations;
+        std::cout << " " << std::setw(9) << t / n_iterations;
```
Comment (peterrum) [inline common_code/benchmark.h:271]: In contrast to your code, I have not summed up the values over all processes (and simply take the values from rank 0). Any opinions here, @kronbichler?

**Review by peterrum — APPROVED** [review]

File: common_code/benchmark.h (hunk 1 above)
Comment (kronbichler) [reply common_code/benchmark.h:271]: Can you run a short comparison between the two? The average could be more realistic as it includes data from all processes and more easily keeps track of some processes running ahead. On the other hand, every CG iterations does an `MPI_Allreduce` so the runahead is naturally limited and one could argue that the difference between the two is mostly noise.

Comment (kronbichler) [reply common_code/benchmark.h:271]: More precisely, as long as the local vector sizes are load-balanced reasonably well, I would expect the difference to be small.

Comment (peterrum) [reply common_code/benchmark.h:271]: > More precisely, as long as the local vector sizes are load-balanced reasonably well, I would expect the difference to be small.

I would say process 0 always should have the most dofs. This is the policy in deal.II, isn't it. Lower ranks are "preferred" int the DoFHandler-policy. But, I am compiling at the moment the release version of deal.II.

Comment (kronbichler) [reply common_code/benchmark.h:271]: Yes. Indeed I would expect the average in the vector updates to be lower than the time of rank 0 if not all MPI ranks are filled.

**Pull request merged**